packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -168,6 +180,25 @@ function Choice(props: ChoicePropsWithForwardRef): React.Node {
                     opacity: showDimmed ? 0.5 : 1.0,
                 }}
             >
+                <div className="perseus-sr-only">
+                    <label>
+                        <input
+                            type={multipleSelect ? "checkbox" : "radio"}
+                            name={questionId}
+                            checked={checked}
+                            onChange={() => {
+                                // If we're checking a crossed-out option, let's
+                                // also uncross it.
+                                sendChange({
+                                    checked: !checked,
+                                    crossedOut: false,
+                                });
+                            }}
+                            disabled={disabled}
+                        />
+                        {a11yText} &nbsp; {content}
```

Comment:
Just as a learning opportunity (NAN), these are some options to avoid the `&nbsp;`

```suggestion
                        {`${a11yText} ${content}`}
```
```suggestion
                        {a11yText + " " + content}
```

Reply:
Thank you! Yes, Jeanette suggested using the `&nbsp;` here; had not used it before this. I think I usually lean towards the interpolation option, though I sometimes use the concatenation option too when I don't want to type the back tick and dollar signs, lol. Is one of the options you suggested the more preferred/cleaner way for combining content? (If this were in Python, I would say more Pythonic, but I don't know if there is a similar expression for React/Javascript.)